### DISCUSSION TITLE
New city: Fremantle, Western Australia

### DISCUSSION THREAD

**Discussion by samwilson** [EqualStreetNames/equalstreetnames#763] [New city]
I've prepared the template for Fremantle: https://github.com/Wikimedia-AU/equalstreetnames-fremantle

**Comment by jbelien** [comment]
Welcome @samwilson ! 🎉 
You should have received an invite to join @EqualStreetNames/fremantle team.
Once you've joined the team, you'll be able to transfer ownership of your repo to @EqualStreetNames organization and I'll take care of the next steps. 👍

**Comment by samwilson** [comment]
Great, thanks! It's now at https://github.com/EqualStreetNames/equalstreetnames-fremantle

**Comment by jbelien** [comment]
Thanks!
Everything is ready on my side ; you just need to add your `MAPBOX_TOKEN` in the repository secrets.

**Comment by samwilson** [comment]
Done.

**Comment by jbelien** [comment]
🚀 Deployed: https://fremantle.equalstreetnames.org/

**Comment by samwilson** [comment]
Great, thank you!! :-) Looks terrific.

We updated the Wikidata info about a street, but it's not appearing on the map. The street is Shuffrey Street, here: https://fremantle.equalstreetnames.org/en/index.html#18.28/-32.048539/115.75176 — do you have any idea why it's not appearing? It seems like it's including it in the queries, and the data is [correct on Wikidata](https://www.wikidata.org/wiki/Q55548351).

**Comment by jbelien** [comment]
> We updated the Wikidata info about a street, but it's not appearing on the map. The street is Shuffrey Street, here: https://fremantle.equalstreetnames.org/en/index.html#18.28/-32.048539/115.75176 — do you have any idea why it's not appearing? It seems like it's including it in the queries, and the data is [correct on Wikidata](https://www.wikidata.org/wiki/Q55548351).

I've investigated why that street doesn't show up.
Here is why: https://github.com/EqualStreetNames/equalstreetnames-fremantle/actions/runs/5495858771/jobs/10015506467#step:7:55
> No `role="street"` member in relation(9361896).

**Comment by samwilson** [comment]
Ah! great, thank you! I'll fix it up (and any others like it).
